add api doc for placement/placementdecision

elgnay opened the pull request:
Signed-off-by: Yang Le <yangle@redhat.com>

elgnay committed:
add api doc for placement/placementdecision

Signed-off-by: Yang Le <yangle@redhat.com>

elgnay commented:
/assign @dockerymick

dockerymick commented on apis/placement.json.adoc:
```diff
@@ -0,0 +1,315 @@
+
+[options="header", cols=".^3a,.^4a"]
+|===
+|Name|Schema
```
Please add a description column for the Definitions

elgnay replied:
Added.

dockerymick commented on apis/placementdecision.json.adoc:
```diff
@@ -0,0 +1,257 @@
+
+[options="header", cols=".^3a,.^4a"]
+|===
+|Name|Schema
```
Same suggestion here, please add a Description  column

elgnay replied:
Added.

dockerymick committed:
Update placement.json.adoc

Few updates

dockerymick committed:
Update placementdecision.json.adoc

changes for consistency

dockerymick committed:
Update placement.json.adoc

updating the title so that it is consistent with the the title used for the reference link

dockerymick committed:
Update placementdecision.json.adoc

Updating the title name for consistency

dockerymick committed:
Update main.adoc

Adding files to the navigation

dockerymick reviewed (approved):
@elgnay thanks for adding the description column and for your updates. I made a couple of commits for consistency and added the files to the navigation so that the build can render. Moving forward to merge this

dockerymick commented:
Related doc issue: https://github.com/open-cluster-management/backlog/issues/13762
